modular_skyrat/modules/credits/end.dm

```diff
@@ -0,0 +1,192 @@
+		possible_titles += "THE CREW LEARNS ABOUT [pick("LOVE", "DRUGS", "THE DANGERS OF MONEY LAUNDERING", "XENIC SENSITIVITY", "INVESTMENT FRAUD", "KELOTANE ABUSE", "RADIATION PROTECTION", "SACRED GEOMETRY", "STRING THEORY", "ABSTRACT MATHEMATICS", "[pick("UNATHI", "SKRELLIAN", "DIONAN", "KHAARMANI", "VOX", "SERPENTID")] MATING RITUALS", "ANCIENT CHINESE MEDICINE")]"
+		possible_titles += "A VERY [pick("CORPORATE", "NANOTRASEN", "FLEET", "HAPHAESTUS", "DAIS", "XENOLIFE", "EXPEDITIONARY", "DIONA", "PHORON", "MARTIAN", "SERPENTID")] [pick("CHRISTMAS", "EASTER", "HOLIDAY", "WEEKEND", "THURSDAY", "VACATION")]"
+		possible_titles += "[pick("GUNS, GUNS EVERYWHERE", "THE LITTLEST ARMALIS", "WHAT HAPPENS WHEN YOU MIX MAINTENANCE DRONES AND COMMERCIAL-GRADE PACKING FOAM", "ATTACK! ATTACK! ATTACK!", "SEX BOMB", "THE LEGEND OF THE ALIEN ARTIFACT: PART [pick("I","II","III","IV","V","VI","VII","VIII","IX", "X", "C","M","L")]")]"
+		possible_titles += "[pick("SPACE", "SEXY", "DRAGON", "WARLOCK", "LAUNDRY", "GUN", "ADVERTISING", "DOG", "CARBON MONOXIDE", "NINJA", "WIZARD", "SOCRATIC", "JUVENILE DELIQUENCY", "POLITICALLY MOTIVATED", "RADTACULAR SICKNASTY")] [pick("QUEST", "FORCE", "ADVENTURE")]"
+		possible_titles += "[pick("THE DAY [pick("NANOTRASEN", "THE SYNDICATE", "ARMADYNE CORPORATION")] STOOD STILL", "HUNT FOR THE GREEN WEENIE", "ALIEN VS VENDOMAT", "SPACE TRACK")]"
+		titles += "<center><h1>EPISODE [rand(1,1000)]<br>[pick(possible_titles)]<h1></h1></h1></center>"
+	else
+		titles += "<center><h1>EPISODE [rand(1,1000)]<br>[GLOB.end_credits_title]<h1></h1></h1></center>"
+
+	for(var/datum/mind/mind in get_crewmember_minds())
+		if(!cast.len && !chunksize)
+			chunk += "CAST:"
+		var/datum/job/job = mind?.assigned_role
+		var/used_name = mind?.current.name
+		var/antag_string
+		for(var/datum/antagonist/antagonist as anything in mind?.antag_datums)
+			antag_string ? (antag_string += ", ") : (antag_string += "...")
+			antag_string += "[antagonist.name]"
+		chunk += "[used_name]\t \t THE \t \t[mind?.antag_datums ? "[antag_string] AND [job.title]" : job.title]"
+		for(var/datum/objective/objective as anything in mind.get_all_objectives())
+			chunk += "<B>[objective.objective_name] </B>: [objective.explanation_text]"
+
+
+
+		chunksize++
+		if(chunksize > 2)
+			cast += "<center>[jointext(chunk,"<br>")]</center>"
+			chunk.Cut()
+			chunksize = 0
+	if(chunk.len)
+		cast += "<center>[jointext(chunk,"<br>")]</center>"
+
+	titles += cast
+
+	var/list/corpses = list()
+	var/list/monkies = list()
+	for(var/mob/living/carbon/human/H in GLOB.dead_mob_list)
+		if(H.timeofdeath < 5 MINUTES) //no prespawned corpses
```

Comment:
are we sure this will do what the comment says? is it not possible it will filter out REALLY early on deaths?